pkg/tfbridge/schema_test.go

```diff
@@ -828,9 +828,14 @@ func TestCustomTransforms(t *testing.T) {
 		nil, "v", resource.PropertyValue{}, resource.NewObjectProperty(resource.NewPropertyMapFromMap(doc)),
 		tfs, psi, nil, nil, false, false)
 	assert.NoError(t, err)
-	if !assert.Equal(t, `{"a":99,"b":false}`, v1) {
-		assert.Equal(t, `{"b":false,"a":99}`, v1)
-	}
+	assert.Equal(t, `{"a":99,"b":false}`, v1)
```

Comment:
No. `TransformJSONDocument` uses [`json.Marshal`](https://golang.org/pkg/encoding/json/#Marshal), which sorts the keys:

> The map keys are sorted and used as JSON object keys

The previous map-order dependent check wasn't necessary, and it doesn't even work. I tried switching the order of the checks and the whole test will fail if the first `assert.Equal` fails; it won't proceed inside the `if`. Since it's not needed and doesn't work as intended, I simplified the previous check.